### ISSUE TITLE
Provide a way to accept a list of tests

### ISSUE THREAD

**Issue by LouisTrezzini** [sebastianbergmann/phpunit#2253]
## TL; DR

Add `phpunit [options] UnitATest.php [UnitBTest.php ... UnitZTest.php]`
## Explanation

It would be great if PHPUnit could accept a list a of tests to run.

A use case could be CircleCI parallel tests. If we provide such a test block in `circle.yml`

```
test:
  override:
    - phpunit:
        parallel: true
        files:
          - src/*/*Bundle/Tests/**/*Test.php
```

CircleCI will split the tests (intelligently if we provide JUnit logs) and launch such commands on separate containers:

```
phpunit 'src/Foo/BarBundle/Tests/ATest.php' 'src/Foo/BarBundle/Tests/BTest.php' 'src/Foo/BarBundle/Tests/CTest.php' 'src/Foo/BarBundle/Tests/DTest.php'
```

Currently, only the first test is run so it's basically useless. I have to split my tests into test suites manually and set each of them to a distinct container if I want to parallelize to speed up my tests.

I may try to provide a PR but I've never contributed to an open source project. I guess it's best to ask what you guys think first.

About the syntax, the documentation is currently unclear as it states 

```
phpunit [options] UnitTest [UnitTest.php]
phpunit [options] <directory>
```

But `phpunit [options] UnitTest.php` works. I could simply add a forth way which wouldn't be compatibility breaking `phpunit [options] UnitATest.php [UnitBTest.php ... UnitZTest.php]`

Thanks for your reply

**Comment by sebastianbergmann** [comment]
I would consider a pull request that implements but will not work on this myself.

**Comment by johnbillion** [comment]
Related: #3387

**Comment by sebastianbergmann** [comment]
Superseded by #3387.
